### DISCUSSION TITLE
Improve style - Border to a PDF

### DISCUSSION THREAD

**Discussion by sistemasnow** [YetiForceCompany/YetiForceCRM#15794] [How to]
Hi Yeti,

I am trying to generate a pdf file from Helpesk module with a sentence like this:

$(recordsList : HelpDesk|ticket_no,ticket_title,ticketstatus,attention||214)$

It runs but the style its so poor. I would like to improve the style options like a grid, as if it was a table with border. I don´t know if is possible to do it. Perhaps, modifiying some php or similar...

Can you help me?

**Comment by mariuszkrzaczkowski** [comment]
Set params table=border
```
$(recordsList : $moduleName|$columns|$conditions|$viewIdOrName|$limit|$maxLength|$params)$
```
Example
```
$(recordsList : HelpDesk|ticket_no,ticket_title,ticketstatus,attention||214|||table=border)$
```

**Reply by sistemasnow** [reply]
Ummmm, the result is the same. I have attached a file to view the effect.

In other hand, where can I see info about "$params"?

Thanks in advance.

**Comment by sistemasnow** [comment]
Ummmm, the result is the same. I have attached a file to view the effect.

In other hand, where can I see info about "$params"?

Thanks in advance.

[InfDir2022_03_15.pdf](https://github.com/YetiForceCompany/YetiForceCRM/files/8253327/InfDir2022_03_15.pdf)

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/blob/developer/app/TextParser.php#L1058-L1061
What's your CRM version?

**Comment by sistemasnow** [comment]
Hi Mariusz,

I have installed docker 6.1 package.

Regards.

**Reply by mariuszkrzaczkowski** [reply]
It only works from 6.3
